### PR TITLE
Cleanup, helpers & auto reload

### PR DESCRIPTION
**Pull request by elthariel**

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -1,20 +0,0 @@
-
-<!DOCTYPE html>
-<html lang="en">
-	<head>
-		<title>three.js webgl - geometry - cube</title>
-		<meta charset="utf-8">
-		<style>
-			body {
-				margin: 0px;
-				background-color: #000000;
```
Comment (elthariel) [inline client/index.html]: Blahblahblah

**Comment by elthariel** [comment]
:shipit:

**Comment by elthariel** [comment]
Reviewed 14 of 14 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/elthariel/blocks/1)*

**Comment by elthariel** [comment]
Blah blah blah

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/elthariel/blocks/1#-:-K99esxrpvyJQxBfBEnN:379513620)*

File: client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		<style>
+		html, body {
+			overflow: hidden;
+			width: 100%;
+			height: 100%;
+			margin: 0;
+			padding: 0;
+		}
+		#renderCanvas {
+			width: 100%;
```
Comment (elthariel) [inline client/public/index.html]: sadfasdfasdfasd

Comment (elthariel) [inline client/public/index.html]: !fixed

Comment (elthariel) [inline client/public/index.html]: !fix

**Comment by elthariel** [comment]
:+1:

File: client/public/index.html (hunk 2 above)
Comment (elthariel) [inline client/public/index.html]: !fixed

**Comment by Champii** [comment]
:+1:

**Comment by elthariel** [comment]
Huh ?

**Comment by elthariel** [comment]
You are here: https://github.com/elthariel/blocks/pulse

**Comment by Champii** [comment]
Ouai mais la t'es tout seul:
https://github.com/elthariel/blocks/graphs/contributors

Et sur la page principale il dit "1 contributor"
https://github.com/elthariel/blocks

Jtrouve ca chelou

**Comment by elthariel** [comment]
Idem

**Pull request merged**